rpcd/etc/openstack_deploy/user_variables.yml

```diff
@@ -89,3 +89,13 @@ tempest_service_available_ceilometer: False
 neutron_neutron_conf_overrides:
   nova:
     endpoint_type: internal
+
+# Disable glance image cache and reconfigure cinder conversion dir outside of a small root partition
+glance_flavor: keystone
+glance_glance_api_conf_overrides:
+  DEFAULT:
+    image_cache_dir: None
```

Comment:
This doesn't appear to be required, updating glance_flavour should be sufficient. There are other image cache related variables so I'm not sure there is a benefit to just overriding image_cache_dir.